packages/api_bo/src/resolvers/query.ts

```diff
@@ -1,5 +1,29 @@
+import { ObjectId } from "https://deno.land/x/web_bson@v0.2.3/mod.ts";
+import { Context } from "../app.ts";
+import { CenterModel } from "../models/models.ts";
+
 export const Query = {
-  hello: (): string => {
-    return "hello world";
+  getCenters: async (_parent: unknown, _args: unknown, ctx: Context) => {
+    const db = ctx.db;
+    const centerCollection = db.collection<CenterModel>("centers");
+    const centers = await centerCollection.find().toArray();
+    return centers;
+  },
+  getCenter: async (_parent: unknown, args: { id: string }, ctx: Context) => {
+    const db = ctx.db;
+    const centerCollection = db.collection<CenterModel>("centers");
+    const center = await centerCollection.findOne({
+      _id: new ObjectId(args.id),
+    });
+    if (!center) {
+      throw new Error("Center not found");
+    }
+    return center;
+  },
+};
+
+export const Center = {
+  id: (parent: CenterModel): string => {
```

Comment:
Dentro del tipo Center de graphql tienes campos que no son lo que se guarda en base de datos, por ejemplo los grupos. En la base de datos se guardarán los IDs y tu al front le quieres pasar el tipo Grupo, por lo que aquí tendrás que meter todos esos campos que se guardan distintos de lo que se mandan. Igual para cuando hagas los resolvers de los Monitores o los Grupos, tendrás campos que tienes que resolver aquí a mano.
En este caso, para los grupos debería ser algo así: `groups: async (parent) => GroupCollection.find({_id: {$in: parent.groups}}).toArray()`